newIDE/app/src/SceneEditor/index.js

```diff
@@ -1358,84 +1358,90 @@ export default class SceneEditor extends React.Component<Props, State> {
         )}
         <I18n>
           {({ i18n }) => (
-            <InfoBar
-              show={this.state.showAdditionalWorkInfoBar}
-              identifier={this.state.additionalWorkInfoBar.identifier}
-              message={i18n._(this.state.additionalWorkInfoBar.message)}
-              touchScreenMessage={i18n._(
-                this.state.additionalWorkInfoBar.touchScreenMessage
-              )}
-            />
+            <div>
+              <InfoBar
+                show={this.state.showAdditionalWorkInfoBar}
+                identifier={this.state.additionalWorkInfoBar.identifier}
+                message={i18n._(this.state.additionalWorkInfoBar.message)}
+                touchScreenMessage={i18n._(
+                  this.state.additionalWorkInfoBar.touchScreenMessage
+                )}
+              />
+              <ContextMenu
+                ref={contextMenu => (this.contextMenu = contextMenu)}
+                buildMenuTemplate={() => [
+                  {
+                    label: i18n._(
+                      this.state.selectedObjectNames.length
+                        ? 'Add an Instance of ' +
+                            shortenString(this.state.selectedObjectNames[0], 7)
+                        : ''
+                    ),
```

Comment:
This won't work, you must pass a `t` macro to `i18n._()`, not a string:
```suggestion
                    label: this.state.selectedObjectNames.length
                        ? i18n._(t`Add an Instance of ${shortenString(this.state.selectedObjectNames[0], 7)}`)
                        : ''
                    ),
```

Rule of thumb: i18n._ is always called with `t` macro, and variables inside `${}`. Never any logic inside.